docs/adr/0006-rapid-releases-and-moving-towards-a-monorepo.md

```diff
@@ -0,0 +1,123 @@
+# Rapid releases and moving towards a monorepo
+
+* Status: proposed
+* Deciders:
+  * Sync team: ✅ bendk, tarikeshaq, lougeniaC64, ddurst, mhammond
+  * Release management: rvandermeulen
+  * Release Engineering: cyrus-bond
+  * Firefox Android:  ✅ csadilek
+  * Firefox iOS: ?
+  * Firefox Desktop: ?
```

Comment:
Cyrus Bond and Sylvestre can be added for desktop. I'll email them to make them aware -- they may want to add others.